db/models.py

```diff
@@ -1,5 +1,13 @@
+from django.utils import timezone
+
+from django.contrib.auth.models import AbstractUser
+
 from django.db import models
 
+from django.core.exceptions import ValidationError
```

Comment:
There is no need to add a blank line between imports that belong to the same import group. Remove redundant blank lines